GoogleForCreators/web-stories-wp
# Video: Pause video when trimming

## Summary
Does the following:
- The video stops while dragging the trim handles and displays the relevant frame.
- The video stops for 1000ms when using the arrow keys for moving the trim handles.
- When the user manually stops the video before using the trim handles, the video won't start playing again after using the trim handles.


## Relevant Technical Choices



## To-do



## User-facing changes

| Before | After |
|-|-|
| ![pause-trim-before](https://user-images.githubusercontent.com/637548/136286601-dec303fe-3a6a-4675-b405-d812bdc8ad9d.gif) | ![pause-trim-after](https://user-images.githubusercontent.com/637548/136286612-1d3c897e-c834-4aa6-bbc5-c1f38ac7312b.gif) |

## Testing Instructions






This PR can be tested by following these steps:

1. Add a video (preferably such a video where the content changes enough to detect different moments in the video)
2. Enter trim mode, ensure the video is playing
3. Now drag the trim handles. Verify that when dragging the trim handles, the video pauses and will also adjust itself to display the frame of where the handle currently is.
4. Let go of the handle, verify the video start looping in the new trim range.
5. Now try the same thing with keyboard, using arrow keys. Again, verify, the video pauses while using the drag handles and that it adjusts the displayed frame according to the handle. When stopping using the keys, the video starts playing again.
6. Now pause the video by clicking on the Stop icon.
7. Verify the same things but this time the video should not start automatically playing when stopping dragging the handles / stopping using the arrow keys.

## Checklist




---



Fixes #9108
